_targets.R

```diff
@@ -23,6 +22,11 @@ GFv1_NHDv2_xwalk_url <- paste0("https://raw.githubusercontent.com/USGS-R/drb-net
 # pulled from https://www.sciencebase.gov/catalog/item/5728d6ace4b0b13d3918a992
 nhd_statsgo_parent_sbid <- '5728d6ace4b0b13d3918a992'
 
+## Depth to bedrock data source
+## original source: http://globalchange.bnu.edu.cn/research/dtb.jsp#download (required form submission to obtain)
+## Clipped drb version already stored in Caldera 1_fetch/in. To successfully run this pipeline, this data must be manually copied locally to 1_fetch/in 
+Shangguan_dtb_cm_250m_clip_path <- '1_fetch/in/Shangguan_dtb_cm_250m_clip2/w001001.adf'
```

Comment:
```suggestion
Shangguan_dtb_cm_250m_clip_path <- '1_fetch/in/Shangguan_dtb_cm_250m_clip/w001001.adf'
```

The .7z file on caldera doesn't have a "2" after "clip" in the path name. When I unzip the file, I need to remove the "2" here in order for the pipeline to build.